test/ConfigProviderTest.php

```diff
@@ -15,17 +15,13 @@
 
 class ConfigProviderTest extends TestCase
 {
-    /**
-     * @var ConfigProvider
-     */
+    /** @var ConfigProvider */
     private $provider;
 
-    protected function setUp() : void
-    {
-        $this->provider = new ConfigProvider();
-    }
```

Comment:
Our tests have `setUp` as the first method in tests across the org, please revert the change and bring them back up for consistency here and in the rest of the PR.

Reply:
corrected